csv/algeria_postcodes_export.csv

```diff
@@ -0,0 +1,3941 @@
+03021,أفلو - حي محمد بوضياف,Aflou Hai Med Boudiaf,أفلو - حي محمد بوضياف,Aflou Med Boudiaf,64,أفلو,Aflou,أفلو,Aflou,03,الأغواط,Laghouat
+03022,حاسي الدلاع,Hassi Dalaa,حاسي الدلاع,Hassi Delaa,75,حاسي الدلاعة,Hassi Delaa,حاسي الرمل,Hassi R'mel,03,الأغواط,Laghouat
+03023,الغيشة,El Ghicha,الغيشة,El Ghicha,71,الغيشة,El Ghicha,الغيشة,El Ghicha,03,الأغواط,Laghouat
+03024,سيدي بوزيد,Sidi Bouzid,سيدي بوزيد,Sidi Bouzid,83,سيدي بوزيد,Sidi Bouzid,أفلو,Aflou,03,الأغواط,Laghouat
+03025,قلطة سيدي سعد,Gueltat Sidi Saad,قلطة سيدي سعد,Gueltat Sidi Saad,73,قلتة سيدي سعد,Gueltat Sidi Saad,قتلة سيدي سعيد,Gueltat Sidi Saad,03,الأغواط,Laghouat
+03026,تاويالة,Taouiala,تاويالة,Taouiala,87,تاويالة,Taouiala,بريدة,Brida,03,الأغواط,Laghouat
+03027,وادي مرة,Oued Mourra,وادي مرة,Oued Mourra,80,وادي مرة,Oued Morra,وادي مرة,Oued Morra,03,الأغواط,Laghouat
+03028,عين سيدي علي,Ain Sidi Ali,عين سيدي علي,Ain Sidi Ali,66,عين سيدي علي,Ain Sidi Ali,قتلة سيدي سعيد,Gueltat Sidi Saad,03,الأغواط,Laghouat
+03030,عين سخونة,Ain Skhouna,عين سخونة,Ain Skhouna,83,سيدي بوزيد,Sidi Bouzid,أفلو,Aflou,03,الأغواط,Laghouat
+03031,شكالة,Chekkala,شكالة,Chekkala,83,سيدي بوزيد,Sidi Bouzid,أفلو,Aflou,03,الأغواط,Laghouat
+03032,برج سنوسي,Bordj Senouci,برج سنوسي,Bordj Senouci,79,الأغواط,Laghouat,الأغواط,Laghouat,03,الأغواط,Laghouat
+03033,بن ناصر بن شهرة,Benacer Benchohra,بن ناصر بن شهرة,Benacer Benchohra,68,بن ناصر بن شهرة,Benacer Benchohra,قصر الحيران,Ksar El Hirane,03,الأغواط,Laghouat
+03034,سبقاق,Sebgag,سبقاق,Sebgag,82,سبقاق,Sebgag,أفلو,Aflou,03,الأغواط,Laghouat
+03035,طرغالل,Terghalel,تركلل,Terkellel,71,الغيشة,El Ghicha,الغيشة,El Ghicha,03,الأغواط,Laghouat
+03036,الأغواط - جمعية,Laghouat Djamia,الأغواط - جمعية,Laghouat Djamiaa,79,الأغواط,Laghouat,الأغواط,Laghouat,03,الأغواط,Laghouat
+03037,أنفوس,Anfous,أنفوس,Anfous,71,الغيشة,El Ghicha,الغيشة,El Ghicha,03,الأغواط,Laghouat
+03038,حاج مشاري,Hadj Mecheri,حاج مشاري,Hadj Mecheri,74,الحاج مشري,Hadj Mechri,بريدة,Brida,03,الأغواط,Laghouat
+03040,الحويطة,El Houita,الحويطة,El Houita,72,الحويطة,El Haouaita,عين ماضي,Ain Madhi,03,الأغواط,Laghouat
+03041,وادي مزي,Oued M'Zi,وادي مزي,Oued M'Zi,81,وادي مزي,Oued M'zi,وادي مرة,Oued Morra,03,الأغواط,Laghouat
+03042,لالماية,Lalmaya,لالماية,Lalmaya,86,تاجرونة,Tadjrouna,عين ماضي,Ain Madhi,03,الأغواط,Laghouat
+03043,وادي الطويل,Oued Touil,وادي الطويل,Oued Touil,73,قلتة سيدي سعد,Gueltat Sidi Saad,قتلة سيدي سعيد,Gueltat Sidi Saad,03,الأغواط,Laghouat
+03044,حسيان الذيب,Hassiane Dhib,حسيان الذيب,Hassiane Dhib,73,قلتة سيدي سعد,Gueltat Sidi Saad,قتلة سيدي سعيد,Gueltat Sidi Saad,03,الأغواط,Laghouat
+03046,بوزباير,Bouzbaeir,بوزباير,Bouzbaeir,76,حاسي الرمل,Hassi R'mel,حاسي الرمل,Hassi R'mel,03,الأغواط,Laghouat
+03047,أفلو - زاوي سعيد,Aflou Zaoui Said,أفلو - زاوي سعيد,Aflou Zaoui Said,64,أفلو,Aflou,أفلو,Aflou,03,الأغواط,Laghouat
+03048,"الأغواط - ""DCA""",Laghouat D C A,"الأغواط - ""DCA""",Laghouat Dca,79,الأغواط,Laghouat,الأغواط,Laghouat,03,الأغواط,Laghouat
```

Comment:
I think things like these double quotes are present in the Json payload too.
These are put in when double quotes are part of the entry and should be looked into as possible flaws for that entry.

Reply:
I think we should stop here and not do any more formatting. I worry we are experiencing mission creep.

I think maybe formatting should be its own separate objective.

Reply:
quotes pertaining to preceding or trailing spaces disappear when these spaces are removed but as stayed above, other scenarios may need to be addressed.